resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/ratelimiter/autoconfigure/AbstractRateLimiterConfigurationOnMissingBean.java

```diff
@@ -80,7 +80,7 @@ public RateLimiterRegistry rateLimiterRegistry(
     public RateLimiterAspect rateLimiterAspect(
         RateLimiterConfigurationProperties rateLimiterProperties,
         RateLimiterRegistry rateLimiterRegistry,
-        @Autowired(required = false) List<RateLimiterAspectExt> rateLimiterAspectExtList,
+        @Autowired(required = false) List<RateLimiterDecoratorExt> rateLimiterAspectExtList,
```

Comment:
Could you change variable name? 
```suggestion
        @Autowired(required = false) List<RateLimiterDecoratorExt> rateLimiterDecoratorExtList,
```